test/parallel/test-inspector-stress-http.js

```diff
@@ -0,0 +1,32 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const { NodeInstance } = require('../common/inspector-helper.js');
+
+async function testHttp(child, number) {
+  try {
+    const response = await child.httpGet(null, '/json/list');
+    console.log(`Interation ${number} done`);
+    return response;
+  } catch (e) {
+    console.error(`Iteration ${number} failed`, e);
```

Comment:
Shouldn't the exception bubble up?  Also, there seems to be no point in returning `response`.